packages/components/src/DateTimePickers/DateRange/Input/Input.component.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import DebounceInput from 'react-debounce-input';
+
+import { DateRangeContext } from '../Context';
+import InputSizer from '../../shared/InputSizer';
+
+function Input(props, ref) {
+	const { date, onChange, onFocus, label } = props;
+	const { inputManagement } = useContext(DateRangeContext);
+	const { placeholder } = inputManagement;
+
+	return (
+		<div>
+			<label htmlFor={props.id} className="control-label">
+				{label}
+			</label>
+			<InputSizer placeholder={placeholder} inputText={date.value ? date.textInput : ''}>
+				{width => (
+					<DebounceInput
+						id={props.id}
+						autoComplete="off"
+						className="form-control"
+						debounceTimeout={300}
+						type="text"
+						placeholder={placeholder}
+						value={date.textInput}
+						style={{ width }}
+						onChange={onChange}
+						onFocus={onFocus}
+						inputRef={ref}
```

Comment:
You don't pass/spread additional props from InputDateRangePicker. That can lead to accessiibility issue?
For example if we want to have an error div, we want to associate the inputs with this error via `aria-describedby`. And there can be tons of cases. Just spread the rest of the props from InputDateRangePicker to DateRange.Input, and from DateRange.Input to the DebounceInput please